packages/client/src/arpa_reporter/router/index.js

```diff
@@ -1,10 +1,11 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 
+import HomeView from '@/arpa_reporter/views/HomeView.vue';
+import LoginView from '@/arpa_reporter/views/LoginView.vue';
+import ValidationView from '@/arpa_reporter/views/ValidationView.vue';
```

Comment:
Any reason that only this subset of imports has been migrated to the `@` prefix?

Reply:
I'll go ahead and merge these as-is (assuming your approval on the PR means you're on board with that), but if you have thoughts on any way you'd rather handle these imports, feel free to say and I'll do some follow-up!